test/wdio/json-basic/cmp.test.tsx

```diff
@@ -0,0 +1,14 @@
+import { h, Fragment } from '@stencil/core';
```

Comment:
Non-blocking - `Fragment` isn't used in this test and can be removed
```suggestion
import { h } from '@stencil/core';
```